src/lib.rs

```diff
@@ -1722,6 +1722,20 @@ impl Build {
                         } else if target.contains("aarch64") {
                             cmd.args.push("--target=aarch64-unknown-windows-gnu".into())
                         }
+                    } else if target.ends_with("-freebsd") {
+                        // clang 13 on FreeBSD doesn't support a target triple without at least the
+                        // major os version number appended; e.g. use x86_64-unknown-freebsd13
+                        // instead of x86-64-unknown-freebsd
+                        let uname_r = String::from_utf8(
+                            std::process::Command::new("uname")
+                                .args(&["-r"])
+                                .output()
+                                .expect("Failed to execute uname!")
```

Comment:
Please return Err here and below rather than unwrap/expect (this function already returns a Result so it shouldn't be too bad).